code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -1247,3 +1247,27 @@
 		return FALSE
 
 	return TRUE
+
+// used in the simulation room for cas runs, removed the sound and ammo depletion methods.
+// copying code is definitely bad, but adding an unnecessary sim or not sim boolean check in the open_fire_firemission just doesn't seem right.
+/obj/structure/dropship_equipment/weapon/proc/open_simulated_fire_firemission(obj/selected_target, mob/user = usr)
+	set waitfor = 0
+	var/turf/target_turf = get_turf(selected_target)
+	var/obj/structure/ship_ammo/SA = ammo_equipped //necessary because we nullify ammo_equipped when firing big rockets
+	var/ammo_accuracy_range = SA.accuracy_range
+	// no warning sound and no travel time
+	last_fired = world.time
+	if(linked_shuttle)
+		for(var/obj/structure/dropship_equipment/electronics/targeting_system/TS in linked_shuttle.equipments)
+			ammo_accuracy_range = max(ammo_accuracy_range-2, 0) //targeting system increase accuracy
+			break
```

Comment:
this could be something like

```
if(locate(/obj/structure/dropship_equipment/electronics/targeting_system) in linked_shuttle.equipments)
 ammo_accuracy_range = max(ammo_accuracy_range - 2, 0)